crates/l2/proposer/prover_server.rs

```diff
@@ -393,64 +531,147 @@ impl ProverServer {
         })
     }
 
-    pub async fn send_proof(
+    pub async fn handle_proof_submission(
```

Comment:
Use constants for the magic numbers.

Reply:
What do you mean by magic numbers?

Reply:
i've implemented [some constants](https://github.com/lambdaclass/ethrex/blob/21fee227606c4ee7277c97ad499b88fc1cc06e53/crates/l2/utils/prover/proving_systems.rs#L36) for PR #1477 which is the following PR to merge.